browser/main/lib/dataApi/attachmentManagement.js

```diff
@@ -245,39 +245,41 @@ function deleteAttachmentFolder (storageKey, noteKey) {
  * @param noteKey NoteKey of the current note. Is used to determine the belonging attachment folder.
  */
 function deleteAttachmentsNotPresentInNote (markdownContent, storageKey, noteKey) {
-  const targetStorage = findStorage.findStorage(storageKey)
-  const attachmentFolder = path.join(targetStorage.path, DESTINATION_FOLDER, noteKey)
-  const attachmentsInNote = getAttachmentsInContent(markdownContent)
-  const attachmentsInNoteOnlyFileNames = []
-  if (attachmentsInNote) {
-    for (let i = 0; i < attachmentsInNote.length; i++) {
-      attachmentsInNoteOnlyFileNames.push(attachmentsInNote[i].replace(new RegExp(STORAGE_FOLDER_PLACEHOLDER + escapeStringRegexp(path.sep) + noteKey + escapeStringRegexp(path.sep), 'g'), ''))
+  if (storageKey != null && noteKey != null && markdownContent != null) {
```

Comment:
It might be better for this to be able to reduce nesting, like below:
```javascript
if (storageKey == null || noteKey == null || markdownContent == null) return
```

Reply:
done